styles.css

```diff
@@ -0,0 +1,48 @@
+body {
```

Comment:
It is not recommended to style using the html selectors. The proper standard is to use classes.